gruntfile.js

```diff
@@ -166,6 +166,8 @@ function shell_run_container() {
       -p 8000:8000 \
       -p 9000:9000 \
       -p 9443:9443 \
+      --net caddy_proxy \
```

Comment:
Isn't that gonna mess up with running the env locally on other devs machines?

Reply:
yeah, this it totally for my test setup, and shouldn't really be there - but no, it should not make a difference, as the host port mapping still happens.

Reply:
I think it will fail, docker run will fail if it can't find the specified network